Small adjustments to the installation instructions (#50)
Ouranosinc/cookiecutter-pypackage: {{cookiecutter.project_slug}}/CONTRIBUTING.rst

Ready to contribute? Here's how to set up ``{{ cookiecutter.project_name }}`` for local development.

> #. First, clone the ``xhydro`` repo locally.

Reviewer: ```suggestion
#. First, clone the ``xhydro`` repo locally.
```
```suggestion
#. First, clone the ``{{ cookiecutter.project_name }}`` repo locally.
```

        # To simply test that the docs pass build checks
        python -m tox -e docs

#. If changes to your branch are made on GitHub, you can update your local branch with:

>     .. code-block:: console
>         git checkout name-of-your-bugfix-or-feature
>         git fetch
>         git pull origin name-of-your-bugfix-or-feature

Author: ```suggestion
    .. code-block:: console
    
        git checkout name-of-your-bugfix-or-feature
        git fetch
        git pull origin name-of-your-bugfix-or-feature
```

    If you have merge conflicts, you might need to replace `git pull` with `git merge` and resolve the conflicts manually.
    Resolving conflicts from the command line can be tricky. If you are not comfortable with this, you can ignore the last command and instead use a GUI like PyCharm or Visual Studio Code to merge the remote changes and resolve the conflicts.

> #. Before merging, your Pull Request will need to be based on the `main` branch of the `xhydro` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:

Author: ```suggestion
#. Before merging, your Pull Request will need to be based on the `main` branch of the {{ cookiecutter.project_name }} repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:
```

> #. Before merging, your Pull Request will need to be based on the `main` branch of the `xhydro` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:

Reviewer: ```suggestion
#. Before merging, your Pull Request will need to be based on the `main` branch of the `xhydro` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:
```
```suggestion
#. Before merging, your Pull Request will need to be based on the `main` branch of the ``{{ cookiecutter.project_name }}`` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:
```

> #. Before merging, your Pull Request will need to be based on the `main` branch of the {{ cookiecutter.project_name }} repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:

Reviewer: ```suggestion
#. Before merging, your Pull Request will need to be based on the `main` branch of the {{ cookiecutter.project_name }} repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:
```
```suggestion
#. Before merging, your Pull Request will need to be based on the `main` branch of the ``{{ cookiecutter.project_name }}`` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:
```

>     .. code-block:: console

Author: ```suggestion
    .. code-block:: console
    
```

        git checkout name-of-your-bugfix-or-feature
        git fetch
        git pull origin main
    See the previous step for more information on resolving conflicts.

> #. To prevent unnecessary testing of branches that are not ready for review, the `xhydro` repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.

Author: ```suggestion
#. To prevent unnecessary testing of branches that are not ready for review, the {{ cookiecutter.project_name }} repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.
```

> #. To prevent unnecessary testing of branches that are not ready for review, the {{ cookiecutter.project_name }} repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.

Author: I've copy-pasted from `xhydro`, but this might not apply here? Is that now by default in the `cookie-cutter`?

Reviewer: This is not the default here. I don't think the majority of cases need this level of control, so feel free to remove it.

> #. To prevent unnecessary testing of branches that are not ready for review, the {{ cookiecutter.project_name }} repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.

Reviewer: ```suggestion
#. To prevent unnecessary testing of branches that are not ready for review, the {{ cookiecutter.project_name }} repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.
```
```suggestion
#. To prevent unnecessary testing of branches that are not ready for review, the ``{{ cookiecutter.project_name }}`` repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.
```

        # To simply test that the docs pass build checks
        python -m tox -e docs

#. If changes to your branch are made on GitHub, you can update your local branch with:

    .. code-block:: console
    
        git checkout name-of-your-bugfix-or-feature
        git fetch
        git pull origin name-of-your-bugfix-or-feature

    If you have merge conflicts, you might need to replace `git pull` with `git merge` and resolve the conflicts manually.
    Resolving conflicts from the command line can be tricky. If you are not comfortable with this, you can ignore the last command and instead use a GUI like PyCharm or Visual Studio Code to merge the remote changes and resolve the conflicts.

#. Before merging, your Pull Request will need to be based on the `main` branch of the ``{{ cookiecutter.project_name }}`` repository. If your branch is not up-to-date with the `main` branch, you can perform similar steps as above to update your branch:

    .. code-block:: console
    
        git checkout name-of-your-bugfix-or-feature
        git fetch
        git pull origin main

>     See the previous step for more information on resolving conflicts.
> 
> #. To prevent unnecessary testing of branches that are not ready for review, the ``{{ cookiecutter.project_name }}`` repository is set up to run tests only when a Pull Request has been "approved" by a maintainer. Similarly, the notebooks within documentation will only be rebuilt when the Pull Request is "approved", or if the Pull Request makes explicit changes to them. As such, additional changes to the Pull Request might be required after the Pull Request is approved to ensure that the tests pass and the documentation can be built.
> 
> #. Once your Pull Request has been accepted and merged to the `main` branch, several automated workflows will be triggered:

Author: ```suggestion
    See the previous step for more information on resolving conflicts.

#. Once your Pull Request has been accepted and merged to the `main` branch, several automated workflows will be triggered:
```
